datahub-web-react/src/app/entity/chart/profile/ChartHeader.tsx

```diff
@@ -16,34 +16,39 @@ export default function ChartHeader({ platform, description, ownership, lastModi
     const entityRegistry = useEntityRegistry();
 
     return (
-        <>
-            <Typography.Title level={5} style={{ color: 'gray' }}>
-                {platform}
-            </Typography.Title>
-            <Typography.Paragraph>{description}</Typography.Paragraph>
-            <Space direction="vertical">
-                <Avatar.Group maxCount={6} size="large">
-                    {ownership &&
-                        ownership.owners &&
-                        ownership.owners.map((owner: any) => (
-                            <Tooltip title={owner.owner.info?.fullName}>
-                                <Link to={`/${entityRegistry.getPathName(EntityType.CorpUser)}/${owner.owner.urn}`}>
-                                    <Avatar
-                                        style={{
-                                            color: '#f56a00',
-                                            backgroundColor: '#fde3cf',
-                                        }}
-                                        src={
-                                            (owner.owner.editableInfo && owner.owner.editableInfo.pictureLink) ||
-                                            defaultAvatar
-                                        }
-                                    />
-                                </Link>
-                            </Tooltip>
-                        ))}
-                </Avatar.Group>
-                {lastModified && <div>Last modified at {new Date(lastModified.time).toLocaleDateString('en-US')}</div>}
+        <Space direction="vertical" size={15}>
+            <Space split={<Divider type="vertical" />}>
+                <Typography.Text style={{ color: 'grey' }}>Chart</Typography.Text>
+                <Typography.Text strong style={{ color: '#214F55' }}>
+                    {platform}
+                </Typography.Text>
             </Space>
-        </>
+            <Typography.Paragraph>{description}</Typography.Paragraph>
+            <Avatar.Group maxCount={6} size="large">
+                {ownership &&
+                    ownership.owners &&
+                    ownership.owners.map((owner: any) => (
+                        <Tooltip title={owner.owner.info?.fullName}>
+                            <Link to={`/${entityRegistry.getPathName(EntityType.CorpUser)}/${owner.owner.urn}`}>
+                                <Avatar
+                                    style={{
+                                        color: '#f56a00',
+                                        backgroundColor: '#fde3cf',
```

Comment:
I'm seeing these hexes repeated around, it might not hurt to put them in a shared constant to make refactoring easier.

Reply:
Yeah these are no good

Reply:
Been trying to at least extract styles out in the component definition